charts/opentelemetry-collector/examples/daemonset-collector-logs/values.yaml

```diff
@@ -7,3 +7,10 @@ config:
   receivers:
     filelog:
       exclude: []
+  service:
+    pipelines:
+      logs:
+        receivers:
+          - otlp
+          - filelog
```

Comment:
Is this a related change?

Reply:
Do you mind moving it to a separate PR for clean git history?

Reply:
Sure

Reply:
reverted